TeraProxy.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+if test "x$(command -v node)" != "x"; then
```

Comment:
```suggestion
if command -v node >/dev/null 2>&1; then
```

`command` will return a non-zero exit code when it can't find the command, which means the condition fails. Not sure if you need all the redirects, but better safe than sorry.

Reply:
Ahh, wasn't aware it returned a non-zero exit code. I really need to pay more attention to that end of man pages.